patch.crates-io.sh

```diff
@@ -20,6 +20,8 @@ fi
 
 set -e
 
+cp "$solana_dir"/rust-toolchain.toml .
```

Comment:
This actually doesn't work for me when I run `./token/twoxtx-setup.sh` locally (although it was okay in CI, for some reason: https://github.com/solana-labs/solana-program-library/actions/runs/4268055882/jobs/7430253561)

Can we move below L26?

Reply:
aa yes ty!